data/data/openstack/bootstrap/variables.tf

```diff
@@ -1,78 +1,17 @@
-variable "base_image_id" {
-  type        = string
-  description = "The identifier of the Glance image for the bootstrap node."
-}
-
-variable "cluster_id" {
-  type        = string
-  description = "The identifier for the cluster."
-}
-
-variable "ignition" {
-  type        = string
-  description = "The content of the bootstrap ignition file."
-}
-
-variable "bootstrap_shim_ignition" {
-  type        = string
-  description = "The content of the ignition file with user ca bundle."
-}
-
-variable "flavor_name" {
-  type        = string
-  description = "The Nova flavor for the bootstrap node."
-}
-
-variable "api_int_ip" {
-  type = string
-}
-
-variable "external_network" {
-  type    = string
-  default = ""
+variable "master_sg_ids" {
+  type        = list(string)
+  default     = ["default"]
```

Comment:
This should not need a default value. The masters stage will always produce a value, even if it may be empty.

Reply:
Fixed.